maestrowf/datastructures/core/study.py

```diff
@@ -336,7 +336,7 @@ def add_step(self, step):
         """
         # Add the node to the DAG.
         self.add_node(step.name, step)
-        LOGGER.info(
+        LOGGER.debug(
```

Comment:
I think this message is still useful, because it's a good catch for the user to be able to review the steps before launching. It might be worth rewording the message to be a little more use-friendly.